src/views/KeyboardShortcuts.vue

```diff
@@ -1,73 +1,77 @@
 <template>
-	<Content app-name="mail">
-		<Navigation />
-		<AppContent>
-			<AppContentDetails>
-				<h2>{{ t('mail', 'Keyboard shortcuts') }}</h2>
+	<Modal>
+		<div
+		class="modal-content"
+		size="large"
+		:title="t('mail', 'Keyboard shortcut')"
+		@close="$emit('close')">
```

Comment:
```suggestion
	<Modal>
		size="large"
		:title="t('mail', 'Keyboard shortcut')"
		@close="$emit('close')">
```

Reply:
:musical_note:  *if you like it then you shoulda put a ``<div/>`` on it* :musical_note: